nvm.sh

```diff
@@ -1907,6 +1908,64 @@ nvm_download_artifact() {
   nvm_echo "${TARBALL}"
 }
 
+nvm_tar_knows_J() {
+  ERR_MSG=$(tar -J 2>&1 | head -1)
+  [ "_$ERR_MSG" != "_tar: unknown option -- J" ]
```

Comment:
interestingly enough, on my Mac, `tar -J` gives `tar: Must specify one of -c, -r, -t, -u, -x`, but `tar --help | grep '\-J'` shows that it's supported.